packages/sdks/snippets/react/src/components/ProductInfo.tsx

```diff
@@ -0,0 +1,35 @@
+type Product = {
```

Comment:
we can export the type from here and use it to type the useState in `ProductEditorial.tsx` -> `<Product | null>`

```suggestion
export type Product = {
```